crates/astria-sequencer/src/crypto.rs

```diff
@@ -0,0 +1,46 @@
+pub(crate) use ed25519_dalek::{
+    Keypair,
+    PublicKey,
+    Signature,
+    Signer,
+    Verifier,
+};
+
+#[cfg(test)]
+mod test {
```

Comment:
Are these tests just testing that a limited subset of an external crate work as anticipated?

Reply:
no, basically for testing we need test accounts, and if they're deterministic it makes our lives a lot easier. this "test" basically just generates the test accounts. I don't really think this should be here, but I'm not sure where they should live exactly, as I don't want to just hard code some keys in without showing where they came from

Reply:
Gotcha, could we put a comment to that effect here? Not sure of a better place either.